spec/descriptions/putMaintenanceConfig.md

```diff
@@ -17,6 +17,33 @@ This is a description for the fields in the request body:
     - **unit**: the unit of time
 - **type**: `ONE_TIME` or `RECURRENT`
 - **rrule**:  for `RECURRENT` mainteance configurations, the RRULE standard from the [iCalendar Spec](https://datatracker.ietf.org/doc/html/rfc5545)
+- **paused**: indicates whether maintenance configuration is paused or not  
+
+**tagFilterExpressionEnabled (OPTIONAL)**: indicates whether tagFilterExpression is used to filter alert notifications  
+**tagFilterExpression (OPTIONAL)**: tag filter expression used to filter alert notifications that will be muted (this field needs to be provided if **tagFilterExpressionEnabled** is set to true) 
+
+### **Scope**
+There are four supported scopes; application perspective, dynamic focus query, tag filter expression, all entities. Below is the configuration corresponding to each scope:
+
+**Application Perspective**  
+dfq: a valid dynamic focus query  
```

Comment:
Should we specify what a valid DFQ should be for application perspective here? Maybe an example?
@chunwahchung @neelbhavsar99

Reply:
Gotcha, I was referring to api users. Since the example is already in the samples its good. Thanks Chun-Wah : )